Errori nel modulo l10n_it_partner_statement_riba

Ciao a tutti, Facendo un porting dei commit della PR nella 16.0  #3949 nella PR #4820 si notano i seguenti due errori 

```
2025-07-01 09:56:46,542 633 ERROR odoo odoo.addons.l10n_it_partner_statement_riba.tests.test_outstanding_statement: ERROR: TestOutstandingStatement.test_show_unsolved_in_report
Traceback (most recent call last):
  File "/__w/l10n-italy/l10n-italy/l10n_it_partner_statement_riba/tests/test_outstanding_statement.py", line 58, in test_show_unsolved_in_report
    move_lines = report["data"][partner.id]["currencies"][invoice.currency_id.id][
KeyError: 2
 

2025-07-01 09:56:49,834 633 ERROR odoo odoo.addons.l10n_it_partner_statement_riba.tests.test_outstanding_statement: FAIL: TestOutstandingStatement.test_show_unsolved_in_xlsx_report
Traceback (most recent call last):
  File "/__w/l10n-italy/l10n-italy/l10n_it_partner_statement_riba/tests/test_outstanding_statement.py", line 116, in test_show_unsolved_in_xlsx_report
    self.fail("Riba column or invoice row not found")
AssertionError: Riba column or invoice row not found
```

Questi errori si verificano anche in locale facendo un test dei moduli l10n_it_vat_registries e l10n_it_partner_statement_riba nella branch 14 

- [x] 14.0 #4829 
- [ ] 16.0 #4874 
- [ ] 18.0

--- a/l10n_it_ricevute_bancarie/tests/riba_common.py
+++ b/l10n_it_ricevute_bancarie/tests/riba_common.py
@@ -1,5 +1,6 @@
 import os
 
+from odoo import fields
 from odoo.tests import common
 from odoo.tools import config
 
@@ -131,17 +132,12 @@ class TestRibaCommon(common.TransactionCase):
         )
 
     def _create_invoice(self):
-        # ----- Set invoice date to recent date in the system
-        # ----- This solves problems with account_invoice_sequential_dates
+        invoice_date = fields.Date.today()
         self.partner.property_account_receivable_id = self.account_rec1_id.id
-        recent_date = (
-            self.env["account.move"]
-            .search([("invoice_date", "!=", False)], order="invoice_date desc", limit=1)
-            .invoice_date
-        )
         return self.env["account.move"].create(
             {
-                "invoice_date": recent_date,
+                "date": invoice_date,
+                "invoice_date": invoice_date,
                 "move_type": "out_invoice",
                 "journal_id": self.sale_journal.id,
                 "partner_id": self.partner.id,
@@ -164,15 +160,12 @@ class TestRibaCommon(common.TransactionCase):
         )
 
     def _create_sbf_invoice(self):
+        invoice_date = fields.Date.today()
         self.partner.property_account_receivable_id = self.account_rec1_id.id
-        recent_date = (
-            self.env["account.move"]
-            .search([("invoice_date", "!=", False)], order="invoice_date desc", limit=1)
-            .invoice_date
-        )
         return self.env["account.move"].create(
             {
-                "invoice_date": recent_date,
+                "date": invoice_date,
+                "invoice_date": invoice_date,
                 "move_type": "out_invoice",
                 "journal_id": self.sale_journal.id,
                 "partner_id": self.partner.id,

